test/test.js

```diff
@@ -1,5 +1,52 @@
+const path = require("path")
 const test = require("tape").test
+const FlyNotify = require("../")
+const defaults = {
+  title: "Fly",
+  message: "Task completed.",
+  icon: path.resolve(__dirname, "..", "fly-logo.png"),
+}
+const plugin = new FlyNotify()
 
-test("fly-notify", function (t) {
- t.end()
+// Adding this to make sure tests pass since we're not checking that the `node-
+// notifier` functionality works.
+plugin.error = function() {}
+plugin.debug = function() {}
+
+test("Testing `fly-notify`", function (t) {
+  t.plan(2)
+  t.ok(plugin.notify, "Fly-Notify `notify` method exists ~~%")
+  t.equal(typeof plugin.notify, "function", "Fly-Notify `notify` method is a function ~~%")
+  t.end()
+})
+
+test("Testing bad options", function(t) {
```

Comment:
@bucaran okay awesome. I like this lean testing idea you want to keep. The gulp tests are insane. When I wrote framework, I meant more convention. Instead of including fly, maybe if we could have something like Fly.util for tests?